CHANGELOG-1.10.md

```diff
@@ -300,6 +300,8 @@ kind: KubeProxyConfiguration
 
 * Some users, especially those with very large clusters, may see higher memory usage by the kube-controller-manager in 1.10. ([#61041](https://github.com/kubernetes/kubernetes/issues/61041))
 
+* Creating multi-cluster ingresses with [kubemci CLI](https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress) fails. This will be fixed in 1.10.1 release. ([GoogleCloudPlatform/k8s-multicluster-ingress#175](https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/175))
```

Comment:
looks good, but reference the Issue# with the actual fix?

Reply:
I want to keep this one as one where users can come and post if they run into issues. We will keep it open.
This one links to the ingress-gce issue

Reply:
okay, I think that's fine.